src/data/versions.json

```
@@ -1,12 +1,12 @@
 {
     "bedrock": {
-        "supported": "1.20.80-1.21.40",
+        "supported": "1.21.40-1.21.44",
```

```suggestion
        "supported": "1.20.80-1.21.44",
```
1.20.80 onwards is still supported; not just 1.21.40